src/cacheDirectory.ts

```diff
@@ -0,0 +1,54 @@
+import { promises as fs } from 'fs';
+import path from 'path';
+
+export const DEFAULT_CACHE_DIRECTORY_NAME = '.j1-integration';
+
+export function getDefaultCacheDirectory() {
+  return path.resolve(process.cwd(), DEFAULT_CACHE_DIRECTORY_NAME);
+}
+
+interface WriteDataToPathInput {
+  cacheDirectory?: string;
+  path: string;
+  data: object;
+}
+
+/**
+ * Helper function for writing arbirary data to a path
+ * relative to the cache directory
```

Comment:
```suggestion
 * Writes data to a path relative to the cache directory.
```

FWIW, I'm not a big fan of calling out the "helper" nature of a function because, the way I see it, all functions are helpers to other functions 😄

Reply:
Will update.